cmd/playground/tasks.go

```diff
@@ -0,0 +1,202 @@
+package playground
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/cenkalti/backoff/v5"
+	"github.com/spf13/cobra"
+
+	"github.com/iximiuz/labctl/api"
+	"github.com/iximiuz/labctl/internal/labcli"
+)
+
+type tasksOptions struct {
+	output   string
+	wait     bool
+	failFast bool
+	timeout  time.Duration
+}
+
+func (opts *tasksOptions) validate() error {
+	if opts.output != "table" && opts.output != "json" && opts.output != "name" && opts.output != "none" {
+		return fmt.Errorf("invalid output format: %s (supported formats: table, json, name, none)", opts.output)
+	}
+
+	return nil
+}
+
+func newTasksCommand(cli labcli.CLI) *cobra.Command {
+	var opts tasksOptions
+
+	cmd := &cobra.Command{
+		Use:   "tasks <play-id>",
+		Short: "List tasks of a running playground",
+		Args:  cobra.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.validate()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return labcli.WrapStatusError(runListTasks(cmd.Context(), cli, args[0], &opts))
+		},
+	}
+
+	flags := cmd.Flags()
+
+	flags.StringVarP(
+		&opts.output,
+		"output",
+		"o",
+		"table",
+		"Output format: table, json, name, none",
+	)
+
+	flags.BoolVar(
+		&opts.wait,
+		"wait",
+		false,
+		"Wait for each task to reach its final state",
+	)
+
+	flags.BoolVar(
+		&opts.failFast,
+		"fail-fast",
+		false,
+		"If any of the tasks fail while waiting, exit with an error immediately",
+	)
+
+	flags.DurationVar(
+		&opts.timeout,
+		"timeout",
+		30*time.Second,
```

Comment:
This should probably not be set by default, and instead, the loop could run until the play terminates (or all tasks reach the final state).